packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -100,7 +101,8 @@ export function updatePackageFile(model: RLCModel, existingFilePath: string) {
   }
   const packageInfo = JSON.parse(packageFile.getFullText());
 
-  if (hasPaging) {
+  // no need to update paging dependency for modular libraries
+  if (hasPaging && model.options?.isModularLibrary !== true) {
```

Comment:
Can we just set hasPaging as false before generate package.json if it's modular library? Or set it as false from the begining if there's no other places require this hasPaging?

Reply:
yeah, i also think of this way. i didn't do this because i hope the paging metadata is still correct. overriding it as false can't correctly reflect the package info. so the intention of this change is that `this package has paging operations but we don't add paging dependency in modular`.

not sure you have any strong preference here?